### PR TITLE
adding deviceToken note

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** added note about parameter not shared across Classic and IE
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-568720](https://oktainc.atlassian.net/browse/OKTA-568720)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/ad7a9911b0fee5b9ce1ecf7c58a29b1a025bf213/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [67](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6345e9c4-d790-42a1-a256-7d2122df9771) | 51  | 68  | 32  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1298a3c5-62cf-4f5e-93c8-b219cb930313?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 1]
```diff
@@ -91,6 +91,9 @@ The context object allows [trusted web applications](#trusted-application) such
 
 > **Note:** Overriding context such as `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication or recovery requests with a valid *administrator API token*. If an API token is not provided, the `deviceToken` will be ignored.
 
+> **Note:** If you are using Classic Identity Engine, the `deviceToken` parameter isn't shared between the Classic Identity Engine Authentication API and Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:94]:
```suggestion
> **Note:** If you are using Okta Classic Engine, the `deviceToken` parameter isn't shared between the Classic Engine Authentication API and the Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).
```

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 2]
```diff
@@ -91,6 +91,9 @@ The context object allows [trusted web applications](#trusted-application) such
 
 > **Note:** Overriding context such as `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication or recovery requests with a valid *administrator API token*. If an API token is not provided, the `deviceToken` will be ignored.
 
+> **Note:** If you are using Okta Classic Engine, the `deviceToken` parameter isn't shared between the Classic Engine Authentication API and the Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:94]:
```suggestion
> **Note:** If you are using Okta Classic Engine, the `deviceToken` parameter isn't shared between the Authentication API and the Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/467a8fec469d60647860c037ea2bf5199e80aac0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [67](https://okta.acrolinx.cloud/api/v1/checking/scorecards/834e29a3-4e7f-4b4f-b8b1-4cd7dc4e89fa) | 51  | 68  | 32  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6bd8a180-5d57-4015-b383-9a2fadf00876?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/authn/index.md](https://github.com/okta/okta-developer-docs/blob/429899e6ebe6560e82bcece84c0069d2324c4325/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fauthn%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5b8029e8-0b56-4ff0-895c-25ee355b498e) | 51  | 68  | 32  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.496f25c6-a295-4ff8-aa6d-6f3849dc165c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**